### PR TITLE
Artifactory ha nginx bugfix

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by eldada — APPROVED** [review]

File: stable/artifactory-ha/templates/nginx-deployment.yaml [hunk 1]
```diff
@@ -83,6 +85,19 @@ spec:
...
+            port: 80
+          initialDelaySeconds: 300
+          periodSeconds: 10
+          failureThreshold: 10
+        livenessProbe:
+          httpGet:
+            path: '/artifactory/webapp/#/login'
+            port: 80
+          initialDelaySeconds: 300
+          periodSeconds: 10
```
Comment (rjkernick) [inline stable/artifactory-ha/templates/nginx-deployment.yaml]: Please make these configurable through the values.yaml.

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Review by rjkernick — APPROVED** [review]

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -255,6 +255,18 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.service.pool`   | Artifactory instances to be in the load balancing pool. `members` or `all` | `members`    |
 | `artifactory.externalPort`   | Artifactory service external port                         | `8081`                        |
 | `artifactory.internalPort`   | Artifactory service internal port                         | `8081`                        |
+| `artifactory.livenessProbe.enabled`               | would you like a livessProbed to be enabled             |  `true`                                        |
+| `artifactory.livenessProbe.initialDelaySeconds`  | Delay before liveness probe is initiated  | 180                                                   |
+| `artifactory.livenessProbe.periodSeconds`        | How often to perform the probe            | 10                                                   |
+| `artifactory.livenessProbe.timeoutSeconds`       | When the probe times out                  | 10                                                    |
+| `artifactory.livenessProbe.successThreshold`     | Minimum consecutive successes for the probe to be considered successful after having failed. | 3 |
```
Comment (rjkernick) [inline stable/artifactory-ha/README.md]: The livenessProbe.successThreshold must be 1

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/artifactory-ha/README.md (hunk 2 above)
Comment (jainishshah17) [reply stable/artifactory-ha/README.md]: done

**Comment by jainishshah17** [comment]
@rjkernick  
Test is failing because of : ```I0501 21:51:27.549]   9m		3m		22	default-scheduler			Warning		FailedScheduling	0/5 nodes are available: 5 MaxVolumeCount.```

**Pull request merged**